server.js

```diff
@@ -0,0 +1,34 @@
+import express from 'express';
+import mongoose from "mongoose";
+import cors from 'cors';
+const app = express();
+const PORT = 5000;
+import User from "./models/user.js"
+import Post from "./models/post.js";
+import authRoutes from "./routes/auth.js";
+import postRouter from "./routes/post.js";
+import userRouter from "./routes/user.js";
+
+// password-PhFK3sbXy5WZJeG4
+const dbURL = "mongodb+srv://admin:PhFK3sbXy5WZJeG4@cluster0.nduys.mongodb.net/SocialMediaDB?retryWrites=true&w=majority";
```

Comment:
Backend review -> starting. 

This is a secret key and should not be open to everyone. store it in .env/key.js file and then to .gitignore